src/constants/periods.js

```diff
@@ -2,17 +2,23 @@ import i18n from '@dhis2/d2-i18n';
 
 export const getPeriodTypes = () => [
     { id: 'relativePeriods', name: i18n.t('Relative') },
-    { id: 'Daily', name: i18n.t('Daily') },
-    { id: 'Weekly', name: i18n.t('Weekly') },
-    { id: 'Monthly', name: i18n.t('Monthly') },
-    { id: 'BiMonthly', name: i18n.t('Bi-monthly') },
-    { id: 'Quarterly', name: i18n.t('Quarterly') },
-    { id: 'SixMonthly', name: i18n.t('Six-monthly') },
-    { id: 'SixMonthlyApril', name: i18n.t('Six-monthly April') },
-    { id: 'Yearly', name: i18n.t('Yearly') },
-    { id: 'FinancialOctober', name: i18n.t('Financial year (Start October)') },
-    { id: 'FinancialJuly', name: i18n.t('Financial year (Start July)') },
-    { id: 'FinancialApril', name: i18n.t('Financial year (Start April)') },
+    { id: 'DAILY', name: i18n.t('Daily') },
+    { id: 'WEEKLY', name: i18n.t('Weekly') },
+    { id: 'WEEKLYWED', name: i18n.t('Weekly (Start Wednesday)') },
+    { id: 'WEEKLYTHU', name: i18n.t('Weekly (Start Thursday)') },
+    { id: 'WEEKLYSAT', name: i18n.t('Weekly (Start Saturday)') },
+    { id: 'WEEKLYSUN', name: i18n.t('Weekly (Start Sunday)') },
+    { id: 'BIWEEKLY', name: i18n.t('Bi-weekly') },
+    { id: 'MONTHLY', name: i18n.t('Monthly') },
+    { id: 'BIMONTHLY', name: i18n.t('Bi-monthly') },
+    { id: 'QUARTERLY', name: i18n.t('Quarterly') },
+    { id: 'SIXMONTHLY', name: i18n.t('Six-monthly') },
+    { id: 'SIXMONTHLYAPR', name: i18n.t('Six-monthly April') },
+    { id: 'YEARLY', name: i18n.t('Yearly') },
+    { id: 'FYNOV', name: i18n.t('Financial year (Start November)') },
+    { id: 'FYOCT', name: i18n.t('Financial year (Start October)') },
+    { id: 'FYJUL', name: i18n.t('Financial year (Start July)') },
+    { id: 'FYAPR', name: i18n.t('Financial year (Start April)') },
     { id: 'StartEndDates', name: i18n.t('Start/end dates') },
 ];
 
```

Comment:
Import the ids from here: https://github.com/dhis2/analytics/blob/7de0afe0b16f719cd0949d2f84487de18f4c4871/src/index.js#L233 ?

Reply:
It increases the code complexity, but will change so we follow the same code pattern across the apps.

Reply:
Fixed.